lib/reporter.js

```diff
@@ -214,28 +217,24 @@ var CoverageReporter = function (rootConfig, helper, logger, emitter) {
     coverageMap.merge(result.coverage)
   }
 
-  this.onRunComplete = function (browsers, results) {
+  this.onRunComplete = async function (browsers, results) {
     var checkedCoverage = {}
 
-    reporters.forEach(function (reporterConfig) {
-      browsers.forEach(async function (browser) {
-        var coverageMap = coverageMaps[browser.id]
-
+    for (const reporterConfig of reporters) {
+      await Promise.all(browsers.map(async (browser) => {
+        const coverageMap = coverageMaps[browser.id]
         if (!coverageMap) {
           return
         }
 
-        var mainDir = reporterConfig.dir || config.dir
-        var subDir = reporterConfig.subdir || config.subdir
-        var browserName = browser.name.replace(':', '')
-        var simpleOutputDir = generateOutputDir(browserName, mainDir, subDir)
-        var resolvedOutputDir = path.resolve(basePath, simpleOutputDir)
+        const mainDir = reporterConfig.dir || config.dir
+        const subDir = reporterConfig.subdir || config.subdir
+        const simpleOutputPath = generateOutputPath(basePath, browser.name, mainDir, subDir)
```

Comment:
what makes this "simple"? Wouldn't `outputPath` be clear?

Reply:
done. thanks!